lms/envs/aws.py

```diff
@@ -424,6 +424,11 @@
 
 GRADES_DOWNLOAD = ENV_TOKENS.get("GRADES_DOWNLOAD", GRADES_DOWNLOAD)
 
+###### CORS ###########
+
+CORS_ORIGIN_WHITELIST = AUTH_TOKENS.get('CORS_ORIGIN_WHITELIST', CORS_ORIGIN_WHITELIST)
+CORS_ORIGIN_ALLOW_ALL = AUTH_TOKENS.get('CORS_ORIGIN_ALLOW_ALL', CORS_ORIGIN_ALLOW_ALL)
```

Comment:
What's the behavior if both `CORS_ORIGIN_WHITELIST` and `CORS_ORIGIN_REGEX_WHITELIST` are set?  If we need to whitelist a lot of subdomains, then the regex might be simpler.